src/modules/peek/Peek.UI/MainWindow.xaml.cs

```diff
@@ -18,20 +18,28 @@
 using Windows.Foundation;
 using WinUIEx;
 
+using AppWindow = Microsoft.UI.Windowing.AppWindow;
+using Logger = Peek.Common.Helpers.Logger;
```

Comment:
Where is this used? It looks obsolete.

Reply:
I'm guessing that was Visual Studio adding stuff around automatically :S